juju/juju
# NSX-T Network backing of VM creation

## Description of change

When you bootstrapped a vSphere instance specified the external and primary networks on an NSX-T network port, Juju ignored the Opaque Network type and assumed it was a plain switch creating a VM using this backing type. vSphere would create dummy network port and the bootstrapping would be disconnected from the network.

## QA steps

*Please replace with how we can verify that the change works?*

```sh
juju bootstrap home vsp-controller --default-model lab --config primary-network=<nsx-t int net> --config external-network=<nsx-t ext net> --config datastore=<ds> 
```

## Documentation changes

none

## Bug reference

none
